doc/api/fs.md

```diff
@@ -332,6 +332,33 @@ If one or more `filehandle.read()` calls are made on a file handle and then a
 position till the end of the file. It doesn't always read from the beginning
 of the file.
 
+#### `filehandle.readJSON(options)`
```

Comment:
Small bikeshed but... How about just `filehandle.json()` ?

Reply:
I prefer `.json()`

Reply:
`readJSON` is good imo, we have the time to be explicit here (and avoid confusion with `writeJSON`, which would be the obvious follow-up addition here)

Reply:
For writeJSON, I would honestly just extend write() to accept an arbitrary json serializable value rather than a buffer and have that just work... But that's a separate conversation :)

Reply:
That said, I'm good with readJSON, I'd just prefer the shorter given that it matches patterns we see elsewhere (several frameworks use json(), we see text() and blob() in fetch, etc.